core-services/prow/02_config/openshift/cluster-ingress-operator/_prowconfig.yaml

```diff
@@ -1,3 +1,9 @@
+branch-protection:
+  orgs:
+    openshift:
+      repos:
+        cluster-ingress-operator:
+          unmanaged: true
```

Comment:
This will not disable it, but makes us not touch it

Reply:
ah so I do need the `protect: false` then